AutomationFramework/src/main/resources/iosDevice.json

```diff
@@ -0,0 +1,12 @@
+[
```

Comment:
@deshpandevrushali , do we need this file for android?

Reply:
We are using common framework for both the platforms. This file mention the device details on which we suppose to run our iOS tests and have reference to other files also.
I am planning to work on removing files and references of other platform in second phase once we are done with PR integration.

Reply:
@leodoin I have removed this file and couple of other iOS related files.